.circleci/config.yml

```diff
@@ -0,0 +1,184 @@
+#
+#  Globality autogenerated CircleCI configuration
+#
+#  This file is auto generated with globality-build.
+#  You should not make any changes to this file manually
+#
+#  Any changes made to this file will be overwritten in the
+#  next version of the build.
+#
+#  See: http://github.com/globality-corp/globality-build
+#
+#
+
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: ${AWS_ECR_DOMAIN}/globality-build:fe815e7fe622e84fba3ea7c6f72299c4a052f414
```

Comment:
Using the tag here as temporary workaround until new `globality-build` is released incorporating globality-corp/globality-build#161